spec/views/posts/posts_show_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User post show page', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Tomy', photo: 'https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg',
+                         bio: 'Tom bio')
+    @user2 = User.create(name: 'Lima',
+                         photo: 'https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg',
+                         bio: 'Lilly bio')
+    @post1 = Post.create(author_id: @user1.id, title: 'Hello', text: 'This is my first post')
+    @comment1 = Comment.create(post_id: @post1.id, author_id: @user2.id, text: 'Hi Tom!')
+    @comment2 = Comment.create(post_id: @post1.id, author_id: @user2.id, text: 'Hello Tom!')
+
+    visit user_post_path(@user1, @post1)
+  end
+
+  describe 'Display a specific post' do
+    it 'Should display the title of the post' do
+      expect(page).to have_content(@post1.title)
+    end
+
+    it 'Should display the post author' do
+      expect(page).to have_content(@post1.author.name)
+    end
+
+    it 'Should display user comment counter and likes counter' do
+      expect(page).to have_selector('.right-corner', count: 1)
+    end
```

Comment:
- [ Required ] I think it would be nice if you add the following test as specified in the project requirements, I could not find the following test in your spec file. Please see the screenshot below and rectify it.
   
   - Test that asserts you can see how many likes a post has.

This test only tests the `comments` counter, please note that as per your spec file, the number of likes is `zero`
```ruby
    it 'Should display user comment counter and likes counter' do
      expect(page).to have_selector('.right-corner', count: 1)
    end
```

## Project requirements
![Screenshot from 2023-09-09 01-13-52](https://github.com/MoyasiGinko/Rails-Blog/assets/16392046/6e699b9e-231b-4c87-87d6-8dd9771fbe8c)

Reply:
Hey @Felix45 , I don't know how to solve the problem, can you please help me?